Readme.md

```diff
@@ -237,6 +237,85 @@ Your attributes will be automatically saved when you end the session, but if the
 
 We have wrapped up the above example into a high/low number guessing game skill you can [download here](https://github.com/alexa/skill-sample-nodejs-highlowgame).
 
+### Additional Event Processing
+
+The `OnRequest` event is similar to the `NewSession` event but is called regardless of the value of `event.session.new`. If you define `OnRequest` you cannot define `NewSession`. In fact all events will go to `OnRequest` and you will need to call another emit to have processing continue:
+
+```javascript
+var startGameHandlers = Alexa.CreateStateHandler(states.STARTMODE, {
+
+    'OnRequest': function () {
+        if (this.event.session.new) {
+            this.emit('NewSession'); // Uses the handler in newSessionHandlers
+        } else {
+            // Process request when session is not new
+        }
+        
+    },
+
+    // The OnRequest code above is equivalent to following event using NewSession:
+
+    //'NewSession': function () {
+    //    this.emit('NewSession'); // Uses the handler in newSessionHandlers
+    //},
+```
+
+Another important pattern with `OnRequest` is calling `this.emit(':forward')`:
```

Comment:
Let's say that you need to call a login method on a REST API if it is a new session and call an updateStatus method of the API if it is the continuation of an existing session. This needs to happen regardless of which of 5 intents (IntentOne, IntentTwo, etc.) is called. You would use OnRequest to intercept, a condition on this.event.session.new to determine whether myApi.login or myApi.updateStatus is called, and then :forward to continue the flow to one of the 5 intents (whichever would have been called in OnRequest had not been implemented).

Without this.emit(':forward') you would need to do the following:

let eventName = this.event.request.type;
if (this.event.request.intent) {
  eventName = this.event.request.intent.name;
}
this.emitWithState(eventName);

With :forward, you also have the ability to set options that could affect which events are forwarded to and which are not.